packages/perseus/src/widgets/categorizer/categorizer.tsx

```diff
@@ -17,12 +17,15 @@ import Util from "../../util";
 
 import categorizerValidator from "./categorizer-validator";
 
-import type {Rubric, UserInput} from "./categorizer.types";
 import type {PerseusCategorizerWidgetOptions} from "../../perseus-types";
 import type {PerseusStrings} from "../../strings";
 import type {PerseusScore, WidgetExports, WidgetProps} from "../../types";
+import type {
+    PerseusCategorizerRubric,
+    PerseusCategorizerUserInput,
+} from "../../validation.types";
 
-type Props = WidgetProps<RenderProps, Rubric> & {
+type Props = WidgetProps<RenderProps, PerseusCategorizerRubric> & {
```

Comment:
**thought**: Originally, when Perseus lived in webapp, we built the types using a `Perseus` prefix.  Today with Perseus being in its own repo and npm package, I feel like we could begin moving away from prefixing everything with `Perseus`.  It's unrelated to this PR, but seeing all the `PerseusXyz` types brought it to my mind again.